src/test/resources/bandits/bandits-parameters-1.json

```diff
@@ -0,0 +1,87 @@
+{
+  "updatedAt": "2023-09-13T04:52:06.462Z",
+  "bandits": [
+    {
+      "banditKey": "banner",
+      "model": "falcon",
+      "updatedAt": "2023-09-13T04:52:06.462Z",
+      "modelVersion": "v123",
+      "modelData": {
+        "gamma": 1.0,
+        "coefficients": [
+          {
+            "actionKey": "nike",
+            "intercept": 1.0,
+            "actionNumericCoefficients": [
+              {
+                "attributeKey": "brand_affinity",
+                "coefficient": 1.0,
+                "missingValueCoefficient": -0.1
+              }
+            ],
+            "actionCategoricalCoefficients": [
+              {
+                "attributeKey": "loyalty_tier",
+                "values": [
+                  { "value": "gold", "coefficient": 4.5 },
+                  { "value": "silver", "coefficient": 3.2 },
+                  { "value": "bronze", "coefficient": 1.9 }
+                ],
+                "missingValueCoefficient": 0.0
+              }
+            ],
+            "subjectNumericCoefficients": [
+              {
+                "attributeKey": "account_age",
+                "coefficient": 0.3,
+                "missingValueCoefficient": 0.0
```

Comment:
It'll be good to test cases where `missingValueCoefficient` is null or missing. The regression engine won't return a value here if there are no missing values in the training data.

Reply:
Agreed! In a future PR I want to have a test case where a bunch of these fields are missing and make sure things deserialize ok.